Pointwise.lua

```diff
@@ -31,11 +31,14 @@ function Pointwise:createIODescriptors(input)
    end
 
    local nElem = input:nElement()
-   self.nElem = self.nElem or nElem -- this goes to the second branch only once
-   if self.iDesc and nElem == self.nElem then return end
-   self.nElem = nElem
-   self.iDesc = cudnn.toDescriptor(input:view(1,1,1,nElem))
-
+   self.nElem = self.nElem or nElem
+   if self.iDesc and nElem == self.nElem and
```

Comment:
Doesn't a descriptor hold a pointer to Tensor's data? You don't even know if the tensor you have received as input previously haven't been freed in the meantime.